favicon.md

```diff
@@ -0,0 +1,47 @@
+# Cómo generar un favicon
+
+1. Ir a [Favicon Generator](https://realfavicongenerator.net/)
+
+2. Selecciona la imagen
+
+![selectImage](images/selectImage.png)
+
+3. Especifica los ajustes necesarios
+
+![ajustesFavicon1](images/ajustesFavicon1.png)
+![ajustesFavicon2](images/ajustesFavicon2.png)
+![ajustesFavicon3](images/ajustesFavicon3.png)
+![ajustesFavicon4](images/ajustesFavicon4.png)
+![ajustesFavicon5](images/ajustesFavicon5.png)
+![ajustesFavicon6](images/ajustesFavicon6.png)
+
+4. Ir a **Favicon Generator Options** en la misma página
+y genera el favicon
+
+**Nota:** Asegúrate de realizar los ajustes necesarios
+
+![generarFavicon](images/generarFavicon.png)
+
+5. Descarga el paquete
+
+![descargarFavicon](images/descargarFavicon.png)
+
+**Nota:** Una vez descargado el paquete, extrae los archivos y colócalos
+en la raíz del sitio web. Es decir, colocarlos al mismo nivel del `index.html`
+
+![archivosFavicon](images/archivosFavicon.png)
+
+6. Copia las líneas de código que indican y pégalas en `head` en el `index.html`
+
+```
```

Comment:
Github flavored markdown has a nice way to add syntax highlighting to code blocks. 
[Check out the documentation](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)